src/Destination/Request.php

```diff
@@ -94,6 +94,14 @@ public function getMethod(): Method
     /**
      * @return string
      */
+    public function getMethodIdentifier(): string
+    {
+        return $this->getMethod()->getIdentifier();
```

Comment:
@viniciusss, nesse caso seria melhor usar `$this->destination->getMethod()` ou não tem problema ser desse jeito que está?